SwiftTweaks.xcodeproj/project.pbxproj

```diff
@@ -724,7 +724,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "org.Khan-Academy.SwiftTweaks";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_VERSION = 2.3;
+				SWIFT_VERSION = 3.0;
```

Comment:
Can you put this at the project level rather than repeating it for every target?